pkg/controllers/config/labels.go

```diff
@@ -16,17 +16,26 @@ package config
 
 const (
 	// Well known labels on ExternalEntities so that they can be selected by Antrea NetworkPolicies.
-	ExternalEntityLabelKeyPostfix    = "nephe"
-	ExternalEntityLabelKeyNamespace  = "namespace." + ExternalEntityLabelKeyPostfix
-	ExternalEntityLabelKeyKind       = "kind." + ExternalEntityLabelKeyPostfix
-	ExternalEntityLabelKeyVmName     = "name." + ExternalEntityLabelKeyPostfix
-	ExternalEntityLabelKeyTagPostfix = ".tag." + ExternalEntityLabelKeyPostfix
-	ExternalEntityLabelCloudVPCKey   = "vpc." + ExternalEntityLabelKeyPostfix
+	ExternalEntityLabelKeyTagPrefix    = "tag-"
+	ExternalEntityLabelKeyNamespace    = LabelPrefixNephe + "namespace"
+	ExternalEntityLabelKeyKind         = LabelPrefixNephe + "kind"
+	ExternalEntityLabelKeyOwnerVm      = LabelPrefixNephe + "owner-vm"
+	ExternalEntityLabelKeyOwnerVmVpc   = LabelPrefixNephe + "owner-vm-vpc"
+	ExternalEntityLabelKeyCloudRegion  = LabelPrefixNephe + LabelPrefixCloud + "region"
```

Comment:
do you really need const for LabelPrefixCloud?

Reply:
removed